docs/client/python/pyhive.rst

```diff
@@ -0,0 +1,22 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+
+`PyHive`_
+=========
+
+.. warning:: The document you are visiting now is incomplete, please help kyuubi community to fix it if appropriate for you.
+
+.. _PyHive: https://github.com/dropbox/PyHive
```

Comment:
The forked https://github.com/acryldata/PyHive has better support for Kyuubi/Spark